src/main/java/org/folio/rest/impl/DataImportImpl.java

```diff
@@ -489,6 +490,27 @@ public void getDataImportSplitStatus(Map<String, String> okapiHeaders, Handler<A
       .onComplete(asyncResultHandler); 
     });
     
+  }
+  @Override
+  public void postDataImportAssembleStorageFile(AssembleFileDto entity, Map<String, String> okapiHeaders,
+      Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    vertxContext.runOnContext(v -> {
+      try {
+        LOGGER.debug(
+          "postDataImportAssembleStorageFile:: Assemble Storage File to complete upload {}",
+          entity.getKey()
+        );
+        minioStorageService.completeMultipartFileUpload(entity.getKey(),  entity.getUploadId(),entity.getTags())
+          .map(  completed -> completed ? PostDataImportAssembleStorageFileResponse.respond204() : PostDataImportAssembleStorageFileResponse.respond400WithTextPlain("Failed to assemble Data Import upload file") )
```

Comment:
Here's the solution to that smell:

```suggestion
          .map(  completed -> Boolean.TRUE.equals(completed) ? PostDataImportAssembleStorageFileResponse.respond204() : PostDataImportAssembleStorageFileResponse.respond400WithTextPlain("Failed to assemble Data Import upload file") )
```

This happens since `completed` is an object `Boolean`, rather than the primitive; therefore, it can theoretically be `null`.  Additionally, objects can't normally be tested using traditional `==`/`!=`/etc, so we want to use `.equals`